app/controllers/kaui/tag_definitions_controller.rb

```diff
@@ -14,6 +14,7 @@ def new
   end
 
   def create
+    params[:tag_definition][:applicable_object_types] = params[:tag_definition][:applicable_object_types].values unless params[:tag_definition][:applicable_object_types].blank?
```

Comment:
Did you check if this is compatible with 0.18 (the ability to configure applicable object types was introduced in 0.19)?

Reply:
I will verify.

Reply:
I checked with killbill version 0.18.15 and no errors is returned. But, the applicable object types sent are ignored and all types are added.

Reply:
Yup, that's the expected behavior. Thanks for checking!